.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/kobexamoh/.local/share/virtualenvs/cree-intelligent-dictionary-8u1H7t5v/bin/python"
```

Comment:
Heads up! Please do not commit editor files in your PRs. I've gone ahead and placed `.vscode/` in the gitignore, so no need to worry about that in this PR!

Reply:
oof, thanks for catching that! I thought I had excluded it – my bad 😅